docs/source/basics.rst

```diff
@@ -146,44 +157,40 @@ In stub files we don't care about the function bodies, so we use
 an ellipsis instead.  That ``...`` is three literal dots!
 
 Mypy complains if it can't find a stub (or a real module) for a
-library module that you import. You can create a stub easily; here is
-an overview:
-
-* Write a stub file for the library and store it as a ``.pyi`` file in
-  the same directory as the library module.
-* Alternatively, put your stubs (``.pyi`` files) in a directory
-  reserved for stubs (e.g., ``myproject/stubs``). In this case you
-  have to set the environment variable ``MYPYPATH`` to refer to the
-  directory.  For example::
-
-    $ export MYPYPATH=~/work/myproject/stubs
-
-Use the normal Python file name conventions for modules, e.g. ``csv.pyi``
-for module ``csv``. Use a subdirectory with ``__init__.pyi`` for packages.
-
-If a directory contains both a ``.py`` and a ``.pyi`` file for the
-same module, the ``.pyi`` file takes precedence. This way you can
-easily add annotations for a module even if you don't want to modify
-the source code. This can be useful, for example, if you use 3rd party
-open source libraries in your program (and there are no stubs in
-typeshed yet).
-
-That's it! Now you can access the module in mypy programs and type check
-code that uses the library. If you write a stub for a library module,
-consider making it available for other programmers that use mypy
-by contributing it back to the typeshed repo.
-
-There is more information about creating stubs in the
-`mypy wiki <https://github.com/python/mypy/wiki/Creating-Stubs-For-Python-Modules>`_.
-The following sections explain the kinds of type annotations you can use
-in your programs and stub files.
+library module that you import. You can
+:ref:`create a stub easily <stub-files>`.
 
-.. note::
+Next steps
+**********
+
+If you are in a hurry and don't want to read lots of documentation
+before getting started, here are some pointers to quick learning
+resources:
+
+* Read the :ref:`mypy cheatsheet <cheat-sheet-py3>` (also for
+  :ref:`Python 2 <cheat-sheet-py2>`).
+
+* Read :ref:`existing-code` if you have existing code without annotations.
```

Comment:
Maybe elaborate slightly to make it clear that this page will be helpful mostly if you have a large codebase?